includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -58,29 +58,47 @@ class WC_Stripe_Payment_Method_Configurations {
 	 * @return object|null
 	 */
 	private static function get_primary_configuration( $force_refresh = false ) {
-		if ( ! $force_refresh ) {
+		// Only allow fetching payment configuration once per minute.
+		$fetch_cooldown = defined( 'IS_TESTING' ) ? 0 : get_option( 'wcstripe_payment_method_config_fetch_cooldown', 0 );
```

Comment:
Nit: we should be checking that it is defined and true
```suggestion
		$fetch_cooldown = defined( 'WC_STRIPE_IS_TESTING' ) && WC_STRIPE_IS_TESTING ? 0 : get_option( 'wcstripe_payment_method_config_fetch_cooldown', 0 );
```